src/App.tsx

```diff
@@ -1,142 +1,81 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { Footer } from './components/footer_filter/footer_filter';
+import { Header } from './components/header_filter/header_filter';
+import { Main } from './components/main_todos-list/main_todos-list';
 
-const USER_ID = 0;
+const USER_ID = 10548;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [input, setInput] = useState('');
+  const [status, setStatus] = useState('all');
+
+  async function loadedTodos(f: (USER_ID: number) => Promise<Todo[]>) {
+    const result = await f(USER_ID);
+
+    setTodos(result);
+  }
+
+  useEffect(() => {
+    loadedTodos(getTodos);
+  }, []);
```

Comment:
```suggestion
  async function loadedTodos() {
    const result = await getTodos(USER_ID);

    setTodos(result);
  }

  useEffect(() => {
    loadedTodos();
  }, []);
```
redundant pass function through arguments